src/libs/dutil/inc/thmutil.h

```diff
@@ -611,29 +619,6 @@ DAPI_(BOOL) ThemeSetControlColor(
     );
 
 /********************************************************************
- ThemeStartBillboard - starts a billboard control changing images according
-                       to their interval.
-
- NOTE: iImage specifies the image to start on. If iImage is
-       greater than the number of images, the last image shown
-       will be the start image.
-*******************************************************************/
-DAPI_(HRESULT) ThemeStartBillboard(
```

Comment:
Why was this removed?  What if they don't want the billboard to auto-start and/or they want control over which child to show?